examples/expressions.rs

```diff
@@ -6,8 +6,8 @@ use std::error::Error;
 use std::result::Result;
 use std::process::exit;
 use tensorflow::Code;
-use tensorflow::expr::Compiler;
-use tensorflow::expr::Placeholder;
+#[cfg(feature = "tensorflow_unstable")]
```

Comment:
Please add a comment that explains that this is a workaround for https://github.com/rust-lang/cargo/issues/1570.